doc/source/user_guide/style.ipynb

```diff
@@ -1792,6 +1792,135 @@
     "See the template in the [GitHub repo](https://github.com/pandas-dev/pandas) for more details."
    ]
   },
+  {
```

Comment:
prob don't want to add this into the *already* long guide (i guess i am suggesting to break it up into more sections)

Reply:
maybe I put all the relevant information in the `Styler.to_latex` documentation page (which is where someone would probably look anyway), and set that page to point to the user guide and vice versa for other relevant info.